pages/common/qutebrowser.md

```diff
@@ -0,0 +1,24 @@
+# qutebrowser
+
+> A keyboard-driven, vim-like browser based on PyQt5.
+> More information: <https://qutebrowser.org/>.
+
+- Open qutebrowser with a specified (non-default) storage directory:
+
+`qutebrowser --basedir {{base_directory}}`
+
+- Open a qutebrowser instance with temporary settings:
+
+`qutebrowser --set {{option}} {{value}}`
+
+- Restore a named session of a qutebrowser instance:
+
+`qutebrowser --restore {{session_name}}`
+
+- Choose how to open URLs if there is already a qutebrowser instance running:
```

Comment:
Is this correct?  I'm trying to avoid using "choose" because that sounds slightly ambiguous.
```suggestion
- Launch qutebrowser, opening all links using the specified method:
```

Reply:
Why `links` instead of `URLs`, just a question.

Reply:
No particular reason.  I didn't even mean to do that.  Changed back.